scripts/camino_mocks.mockgen.txt

```diff
@@ -0,0 +1,16 @@
+// TODO @evlekht
+// add this to 'scripts/mocks.mockgen.txt' when mockgen will be able to process this files correctly (some generics issues)
+
+github.com/ava-labs/avalanchego/cache=Cacher=cache/mock_cacher.go
```

Comment:
Those mocks are created manually. Even latest mockgen (0.4.0) is failing on those interfaces.